nbs/waterschadeschatter/04_vergelijk_cfg.py

```diff
@@ -35,10 +14,10 @@
 import os
 import numpy as np
 import matplotlib.pyplot as plt
-import hhnk_schadeschatter.functions.wss_loading as wss_loading
-import hhnk_schadeschatter.functions.wss_calculations as wss_calculations
-import hhnk_schadeschatter.functions.wss_main as wss_main
-import hhnk_schadeschatter.functions.add_to_hrt as hrt_temp
+import hhnk_research_tools.waterschadeschatter.wss_loading as wss_loading
+import hhnk_research_tools.waterschadeschatter.wss_calculations as wss_calculations
+import hhnk_research_tools.waterschadeschatter.wss_main as wss_main
+
 
 importlib.reload(wss_main)
 importlib.reload(wss_loading)
```

Comment:
Hieronder verwijderen: `from pathlib import Path`

Reply:
zou ik nu niet teveel aanpassen. moet nog een keer goed uitgwerkt worden. Is meer projectbestand dan dat het in repo hoort.